File: python-sdk/src/planetscaleapi/models/operations/delete_a_branch.py
from __future__ import annotations
import dataclasses



@dataclasses.dataclass
class DeleteABranchPathParams:
    database: str = dataclasses.field(metadata={'path_param': { 'field_name': 'database', 'style': 'simple', 'explode': False }})
    name: str = dataclasses.field(metadata={'path_param': { 'field_name': 'name', 'style': 'simple', 'explode': False }})
    organization: str = dataclasses.field(metadata={'path_param': { 'field_name': 'organization', 'style': 'simple', 'explode': False }})
    

@dataclasses.dataclass
class DeleteABranchRequest:
    path_params: DeleteABranchPathParams = dataclasses.field()
    

@dataclasses.dataclass
class DeleteABranchResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    

File: go-sdk/pkg/models/operations/deleteabranch.go
package operations

type DeleteABranchPathParams struct {
	Database     string `pathParam:"style=simple,explode=false,name=database"`
	Name         string `pathParam:"style=simple,explode=false,name=name"`
	Organization string `pathParam:"style=simple,explode=false,name=organization"`
}

type DeleteABranchRequest struct {
	PathParams DeleteABranchPathParams
}

type DeleteABranchResponse struct {
	ContentType string
	StatusCode  int
}
